facility-manager.html

```diff
@@ -120,12 +123,25 @@
     like to any user's data, in any group (except annotate in Private groups)
     and Group Owners can do anything to any user's data in the groups they
     own.
+    <a href="https://docs.openmicroscopy.org/latest/omero/sysadmins/admins-with-restricted-privileges.html" class="external-url">
```

Comment:
this will become /restricted-admins.html once https://github.com/openmicroscopy/ome-documentation/pull/1783 is merged